src/mobile/src/libs/store.js

```diff
@@ -87,7 +87,7 @@ export const migrate = (store, restoredState) => {
         const propsToReset = [];
 
         // FIXME: Temporarily needed for node list reset
-        if (restoredState.settings.versions.buildNumber < 32) {
+        if (get(restoredState, 'settings.versions.buildNumber') < 32) {
```

Comment:
Just out of curiosity, what's the difference between directly accessing the property vs. using `get`?

Reply:
[lodash.get](https://lodash.com/docs/4.17.11#get) won't throw on `undefined` properties. `{}.settings.versions.buildNumber` will.

Reply:
Ah, I see